components/centos/centos-k8s-singlenode-setup/README.rst

```diff
@@ -0,0 +1,25 @@
+Getting started with vagrantfile
+-----------------------------------------------------------
+
+* This vagrantfile is for setting up a single node kubernetes setup with ADB Vagrant box.
+* i.e. The master node and node setup in the single machine.
+
+Steps
+-----
+
+* git clone the adb git repo
+* change in to the directory containing this vagrant file
+* do vagrant up
+
+::
+     
+     git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git
+     cd adb-atomic-developer-bundle/components/centos/centos-with-kubernetes
+     vagrant up
```

Comment:
@navidshaikh IMO we should describe the steps for better understanding. Sometime users who are not very experienced might want to know what are the steps do.